users/views.py

```diff
@@ -30,6 +34,40 @@ def post(self, request):
         except json.JSONDecodeError:
             return JsonResponse({'MESSAGE':'JSON_Decode_Error'}, status=400)
 
+class FindPwView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        length = 8
+        string_pool = string.ascii_letters + string.digits
+        auth_num = ''
+
+        for i in range(length):
+            auth_num += random.choice(string_pool)
+
+        try:
+            name = data['name']
+            identification = data['identification']
+            email = data['email']
+
+            if not User.objects.filter(Q(name=name) & Q(identification=identification) & Q(email=email)).exists():
+                return JsonResponse({'MESSAGE':'INVALID_NAME OR IDENTIFICATION OR EMAIL'})
```

Comment:
```suggestion
                return JsonResponse({'MESSAGE':'INVALID_USER'}, status=400)
```
그냥 잘 못된 사용자가  나을것 같네요, status_code 잊지마세요

Reply:
수정 완료 했습니다~